### PR TITLE
Go template function to split string into array of strings.

### PR DESCRIPTION
**Pull request by khudayberdiyev25**
This adds new template function `strSplit` which splits string into array of strings.
This function gives more flexibility for custom custom .gotpl templating for resolvers in codegen templates. This function is intended to help developers who wants to write own .gotpl.

### CODE REVIEW

File: codegen/templates/templates.go [hunk 1]
```diff
@@ -206,6 +206,7 @@ func Funcs() template.FuncMap {
 		"call":               Call,
 		"prefixLines":        prefixLines,
 		"notNil":             notNil,
+		"strSplit":           StrSplit,
```
Comment (alexandear) [inline codegen/templates/templates.go:209]: Do we need to cover this with a test?

Comment (khudayberdiyev25) [reply codegen/templates/templates.go:209]: It's actually calling go standard library's `strings.Split` function only.

I would ask some test scenarios If maintainers ask it.

**Comment by StevenACoffman** [comment]
Is this intended to be used in some of the existing gqlgen codegen templates?

**Comment by khudayberdiyev25** [comment]
> this

Not yet. I had some difficulties to write custom .gotpl for resolvers because of lack of string functions in codegen templates. This function is intended to help developers who wants to write own .gotpl.

**Comment by khudayberdiyev25** [comment]
Yes. As gqlgen enables to use custom templates for resolvers, I am using it in my custom resolver.gotpl.

File: codegen/templates/templates.go (hunk 1 above)
Comment (alexandear) [reply codegen/templates/templates.go:209]: According to a [PR](https://github.com/99designs/gqlgen/blob/master/.github/PULL_REQUEST_TEMPLATE.md) requirement, every new feature should have a test.

It doesn't matter how simple the function's implementation is now, someone may change its implementation during a refactoring, and we won't catch this without a test.

See

**Pull request merged**